sdks/python/apache_beam/ml/rag/enrichment/bigquery_vector_search.py

```diff
@@ -0,0 +1,304 @@
+#
```

Comment:
Could we call this out in CHANGES.md (both the enrichment and ingestion pieces)?